### ISSUE TITLE
TFCT: Oil disappeared from lamp

### ISSUE THREAD

**Issue by agilmore2** [TerraFirmaCraft/TerraFirmaCraft#1513]
https://www.twitch.tv/videos/694447054

https://discordapp.com/channels/432522930610765835/432527966266851339/738187810590294107

I was watching this, and I never saw how the oil disappeared. I've watched it at 0.25 speed, and still don't get it.

The lamp was still cooling when filled, but that doesn't appear to change the result.

**Describe the bug**
Filled a lamp with oil and placed it on a ceiling. As soon as it was placed, it showed as a "Bronze Lamp" not as an Olive Oil Bronze Lamp, so it was empty as soon as it was placed.

I've been attempting to reproduce, but have had no luck. 
1. Created metal lamp
2. Filled lamp with oil
3. Placed lamp on ceiling
4. Attempted to light an empty lamp.

**Meta Info**
 - TFC Version: build 156

**Comment by alcatrazEscapee** [comment]
Copying from the other issue:

> pakratt0013 / Today at 9:56 AM
hey, I'm a bit late to update from Tuesday, but I'm hoping that you guys saw that lanterns are broken?
we tested them pretty extensively for a good portion of the last hour.
turning the server off and on emptied the olive oil from the lantern, whether it was on or off...though when the server loaded, it looked on. I don't know if they just stay on after that, but I'm pretty sure the one I set up last week died at some point.
